### PR TITLE
Adding multiphate recipe

### PR DESCRIPTION
**Pull request by carolzhou**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set version = "0.5" %}
+{% set sha256 = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" %}
```
Comment (bgruening) [inline recipes/meta.yaml]: This file is wrong. It's a copy of https://github.com/bioconda/bioconda-recipes/pull/13872/files#diff-56ec1f76a52dd5b54362e8d356fa2ce4 but in the wrong folder. This is why you get this strange error.

Comment (carolzhou) [reply recipes/meta.yaml]: Thank you. I removed the mis-placed meta.yaml file from recipes/. I also modified the meta.yaml under recipes/multiphate/ and removed the build.sh file. More simple tests have been added to meta.yaml under test: commands:. I hope this is now correct and improved.

**Comment by carolzhou** [comment]
The recent meta.yaml with new sha256 (ending in f501) now works on my local test setup.

**Comment by carolzhou** [comment]
This meta.yaml worked in my local testing environment:  $ conda-build meta.yaml.  Now there (again) seems to be a problem with downloading. ???

**Comment by carolzhou** [comment]
This one succeeds for me when I submit via github.com/carolzhou/multiPhATE_b/ repository to Circleci. Running release multiphate as v0.1.2.tar.gz

**Comment by daler** [comment]
Closing due to inactivity, please re-open if needed.

**Pull request closed without merging**